data/txHandler.go

```diff
@@ -175,9 +189,13 @@ func (handler *TxHandler) postprocessCheckedTxn(wi *txBacklogMsg) {
 
 	// at this point, we've verified the transaction, so we can safely treat the transaction as a verified transaction.
 	verifiedTxGroup := wi.unverifiedTxGroup
+	verifyParams := make([]verify.Params, len(wi.verifyContexts))
+	for i := range verifyParams {
+		verifyParams[i] = wi.verifyContexts[i].Params
+	}
```

Comment:
I'm a bit puzzled around this one. It looks like you're pulling the `Params` variable from the `verifyContexts` array, but it doesn't looks like you ever initialized it in `prepareVerifyContexts`. Is it being initialized elsewhere, or did you intended to pass on the empty structure.